Crash while scrolling the log window

```
org.eclipse.swt.SWTException: Failed to execute runnable (java.lang.NullPointerException)
	at org.eclipse.swt.SWT.error(Unknown Source)
	at org.eclipse.swt.SWT.error(Unknown Source)
	at org.eclipse.swt.widgets.Synchronizer.runAsyncMessages(Unknown Source)
	at org.eclipse.swt.widgets.Display.runAsyncMessages(Unknown Source)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Unknown Source)
	at org.eclipse.jface.window.Window.runEventLoop(Window.java:818)
	at org.eclipse.jface.window.Window.open(Window.java:794)
	at com.google.gapid.Main$UI.show(Main.java:118)
	at com.google.gapid.Main.main(Main.java:73)
Caused by: java.lang.NullPointerException
	at com.google.gapid.views.LogView.updateTree(LogView.java:163)
	at com.google.gapid.widgets.Widgets.ifNotDisposed(Widgets.java:120)
	at com.google.gapid.widgets.Widgets.lambda$scheduleIfNotDisposed$1(Widgets.java:148)
	at com.google.gapid.widgets.Widgets.lambda$schedule$0(Widgets.java:140)
	at org.eclipse.swt.widgets.RunnableLock.run(Unknown Source)
	... 7 more
```

--- a/gapic/src/main/com/google/gapid/views/LogView.java
+++ b/gapic/src/main/com/google/gapid/views/LogView.java
@@ -159,7 +159,7 @@ public class LogView extends Composite implements Tab {
           locationToTreeItem(root, severity, location);
         }
       }
-      while (tree.getItemCount() > MAX_ITEMS) {
+      while (tree.getItemCount() > MAX_ITEMS && tree.getTopItem() != null) {
         tree.getTopItem().dispose();
       }
     }

--- a/gapic/src/main/com/google/gapid/views/LogView.java
+++ b/gapic/src/main/com/google/gapid/views/LogView.java
@@ -159,8 +159,8 @@ public class LogView extends Composite implements Tab {
           locationToTreeItem(root, severity, location);
         }
       }
-      while (tree.getItemCount() > MAX_ITEMS && tree.getTopItem() != null) {
-        tree.getTopItem().dispose();
+      while (tree.getItemCount() > MAX_ITEMS) {
+        tree.getItem(0).dispose();
       }
     }
     // Too many new messages to display!

